integration_tests/tests/daphne.rs

```diff
@@ -58,26 +53,20 @@ async fn janus_daphne() {
 
     // Start servers.
     let network = generate_network_name();
-    let (mut task_parameters, leader_task, helper_task) =
-        test_task_builders(VdafInstance::Prio3Count, QueryType::TimeInterval);
+    let (mut task_parameters, task_builder) =
+        test_task_builder(VdafInstance::Prio3Count, QueryType::TimeInterval);
 
     // Daphne is hardcoded to serve from a path starting with /v04/.
-    task_parameters.endpoint_fragments.helper_endpoint_path = "/v04/".to_string();
-    let [leader_task, helper_task]: [Task; 2] = [leader_task, helper_task]
-        .into_iter()
-        .map(|task| {
-            let mut helper_aggregator_endpoint = task.helper_aggregator_endpoint().clone();
-            helper_aggregator_endpoint.set_path("/v04/");
-            task.with_helper_aggregator_endpoint(helper_aggregator_endpoint)
-                .build()
-        })
-        .collect::<Vec<_>>()
-        .try_into()
-        .unwrap();
+    task_parameters.endpoint_fragments.leader_endpoint_path = "/v04/".to_string();
```

Comment:
I think this needs to change as well, so the collector doesn't get a 404 when talking to the Janus leader.